Makefile

```diff
@@ -1,22 +1,17 @@
 .PHONY: all
-all: bindings mocks contracts
+all: provwasm-std contracts
 
-.PHONY: bindings
-bindings:
-	@make -C packages/bindings
-
-.PHONY: mocks
-mocks:
-	@make -C packages/mocks
+.PHONY: provwasm-std
+provwasm-std:
+	@make -C packages/provwasm-std
 
 .PHONY: contracts
 contracts:
 	@make -C contracts
 
 .PHONY: clean
 clean:
-	@make -C packages/bindings clean
-	@make -C packages/mocks clean
+	@make -C packages/provwasm-std clean
 	@make -C contracts clean
```

Comment:
Is it possible to have a make target that calls update-and-rebuild.sh for us? Might make it easier for people to know about that script. Would we want to tie it into `Make all` or would that slow it down too much?

Reply:
Yeah, that's the next work to do, right now there were a couple manual changes to the generated types so I'll need to get that moved into the generator code and then we can automate everything. but won't be part of this PR